### DISCUSSION TITLE
What's the expected time of the waitlist?

### DISCUSSION THREAD

**Discussion by qinyihao** [community/community#3680] [Codespaces] [answered]
Hi there, I joined the Codespaces waitlist in about June last year, and it's already 10 months, but I didn't receive a single message about that. Can you make me into the beta? Thanks a lot.

**Comment by 2percentsilk** [comment, marked as answer]
Thanks for your feedback. We know there's a lot of excitement for the beta (and we're very excited to share it with you!) but we're being cautious with our rollout right now so I ask that you hang tight. We'll add more folks as soon as we can!

**Reply by Damato** [reply]
I was hoping on going paid for the old Visual Studio Code spaces for a new venture, then the whole switchover happened. Everything moved out of planning phase and now we are here. 

There is no alternative except for git-pod and other non oficial avenues.  All roads now point here. 

Yet, like many I have been on the waiting list for ages, and I am still willing to pay - but alas there seems to be no light at the end of the tunnel. Sad.

**Reply by austinsonger** [reply]
@Damato yeah and because Github is taking so long with codespaces, Gitpod has had plenty of time of improving their solution and gitpod is quite easy to use and I will probably continuing using them (due to familiarly now) and due to the fact that they are small company and be better to support them. Github should of just purchased gitpod and incorporated into github... the path of least resistance lol.

**Comment by austinsonger** [comment]
Yeah it feels like I've been waiting forever.

**Comment by chrisbrocklesby** [comment]
I would love to beta this... been waiting for a while also on both my personal and organisation accounts. :(

I really need to see this soon to make a decision on what Project Management tool we should be using (Jira isn't right for us as a team)

**Reply by Druid-svg** [reply]
> I would love to beta this... been waiting for a while also on both my personal and organisation accounts. :(
>
